docker-compose.yml

```diff
@@ -277,25 +285,37 @@ services:
     restart: always
     depends_on:
       - zookeeper
+      - kafka1
+      - kafka2
     volumes:
       - $PWD/security:/etc/kafka/secrets
     environment:
       SCHEMA_REGISTRY_KAFKASTORE_BOOTSTRAP_SERVERS: "SASL_SSL://kafka1:9091,SASL_SSL://kafka2:9092"
+      SCHEMA_REGISTRY_KAFKASTORE_CONNECTION_URL: "zookeeper:2181"
       SCHEMA_REGISTRY_HOST_NAME: schemaregistry
-      SCHEMA_REGISTRY_LISTENERS: "http://0.0.0.0:8081"
+      SCHEMA_REGISTRY_LISTENERS: "http://0.0.0.0:8081,https://0.0.0.0:8082"
```

Comment:
suggest dropping the http listener altogether

Reply:
Removed with latest commit.